File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let mp  = {
        '--X': -1,
        'X++': 1,
        'X--': -1,
        '++X': 1,
    }
    let x = 0;
    for(let i = 0; i < operations.length; i++) {
        x += mp[operations[i]]
    } 
    return x;
};

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.c
int finalValueAfterOperations(char** operations, int operationsSize) {
    int x = 0;
    for(int i = 0; i < operationsSize; i++) {
        if (strcmp(operations[i], "++X") == 0 || strcmp(operations[i], "X++") == 0) {
            x++;
        } else {
            x--;
        }
    }    
    return x;
}